src/Text/Pandoc/Readers/TikiWiki.hs

```diff
@@ -0,0 +1,663 @@
+{-# LANGUAGE RelaxedPolyRec, FlexibleInstances, TypeSynonymInstances #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE FlexibleContexts #-}
+
+{- |
+   Module      : Text.Pandoc.Readers.TikiWiki
+   Copyright   : Copyright (C) 2017 Robin Lee Powell
+   License     : MIT (but large portions are copied from Text.Pandoc.Readers.TWiki , which is GPLv2)
```

Comment:
License needs to be GPLv2 -- that is the standard license for pandoc, and this is derivative from the TWiki reader which is GPLv2.